### ISSUE TITLE
Child of a child

### ISSUE THREAD

**Issue by marcomagnano** [understrap/understrap-child#264]
Alright, I know I sound crazy, and this – well – could be.

I'm trying to create a child theme based on Understrap-child. Why?
Because I'm creating a Wordpress network where the "first-level" child theme is used for all the blogs, but I need to aggregate all the contents from the blogs in the blog n.1

I already did this a bunch of times in all these years, but in this case, I'm trying to reduce to the minimum possible amount the code to maintain, because the client is not exactly stable in its ideas.

So, basically, the "second-level" child theme would inherit everything but the home page and the archive.php file.

The real question is: is this even possible or I'm just wasting my time?

Anyway, congrats to holger1411 for all this AMAZING work. I love understrap!
